src/components/ProgressBar/ProgressView/styles.ts

```diff
@@ -0,0 +1,24 @@
+import styled from 'styled-components/native';
+import {Color} from 'theme/colors';
+
+type ViewBoxPropsType = {
+  questionStatus: 'default' | 'active' | 'right' | 'error';
+};
+
+export const ViewCheck = styled.View<ViewBoxPropsType>`
+  align-items: center;
+  justify-content: center;
+  border: 2px solid ${Color.Gray};
+  border-radius: 50px;
+  width: 15px;
+  height: 15px;
+  margin: 4px;
+  background-color: ${props =>
+    props.questionStatus === 'active'
+      ? Color.Blue
+      : props.questionStatus === 'right'
+      ? Color.BlueLight
+      : props.questionStatus === 'error'
+      ? Color.Red
+      : Color.GrayMedium};
```

Comment:
lets create object for this purpose. like that: statusColor = {active:Color.blue, ...}